src/doc/builtinplugins.rst

```diff
@@ -1193,6 +1193,94 @@ special ``"oiio:ioproxy"`` attributes (see Sections
 the `set_ioproxy()` methods.
 
 
+|
+
+.. _sec-bundledplugins-jpegxl:
+
+JPEG XL
+===============================================
+
+JPEG XL is a new image format that is designed to be a successor to JPEG
+and to provide better compression and quality. JPEG XL files use the file
+extension :file:`.jxl`. The official JPEG XL format specification and other
+helpful info may be found at: https://jpeg.org/jpegxl/
+
```

Comment:
I just wanted to say: thanks so much for writing the documentation section, this is really helpful.

Reply:
You welcome :)